### ISSUE TITLE
loss didn't decline while eval performance is good.

### ISSUE THREAD

**Issue by NovaMind-Z** [hiwonjoon/tf-frrn#6]
Hi, when I have run 100000 epochs for training steps,  the loss didn't decline obviously(actually it remains at a level from 1000 to 4000). But when I use the model I trained to run the  eval, the IoU scores 0.65, I wonder is there something wrong with the loss function?
